.gitignore

```diff
@@ -1,4 +1,5 @@
 .idea
 node_modules
 /dist
-package-lock.json
\ No newline at end of file
+package-lock.json
+coverage
```

Comment:
package-lock.json: I think it's better not to store it in GitHub, just like composer.lock. So that, when running npm install on a fresh checkout, dependencies from package.json are checked instead of relying on the intermediary lock file.

coverage is added because this folder is generated when running test coverage. This is not data we want to store on GitHub.